fifteen_min/src/isochrone.rs

```diff
@@ -113,16 +120,26 @@ impl Isochrone {
             return None;
         }
 
-        let req = PathRequest::between_buildings(
-            map,
-            self.start,
-            to,
-            match self.options {
-                Options::Walking(_) => PathConstraints::Pedestrian,
-                Options::Biking => PathConstraints::Bike,
-            },
-        )?;
-        map.pathfind(req).ok()
+        let constraints = match self.options {
+            Options::Walking(_) => PathConstraints::Pedestrian,
+            Options::Biking => PathConstraints::Bike,
+        };
+
+        let all_paths: Vec<Path> = self
+            .start
+            .iter()
+            .map(|b_id| {
+                map.pathfind(PathRequest::between_buildings(map, *b_id, to, constraints).unwrap())
+                    .ok()
+                    .unwrap()
+            })
+            .collect();
+
+        all_paths
+            .iter()
```

Comment:
`iter()` will borrow each thing in the vector, which is why you wind up with `&Path`. If you were to try to use `all_paths` after this expression, it would still be there. `into_iter()` is what you're looking for -- it consumes the vector, so you'll get `Path` when you map over it, and `all_paths` will be destroyed in the process